metadeploy/api/serializers.py

```diff
@@ -213,22 +237,14 @@ def get_preflight_message(self, obj):
     def circumspect_visible(self, obj, user):
         return obj.is_visible_to(user) and obj.version.product.is_visible_to(user)
 
-    def get_is_allowed(self, obj):
+    def get_is_allowed(self, obj) -> bool:
         return obj.is_visible_to(self.context["request"].user)
 
-    def get_not_allowed_instructions(self, obj):
+    def get_not_allowed_instructions(self, obj) -> str:
         if not obj.version.product.is_visible_to(self.context["request"].user):
             return getattr(obj.version.product.visible_to, "description_markdown", None)
         return getattr(obj.visible_to, "description_markdown", None)
 
-    def get_requires_preflight(self, obj):
-        return obj.requires_preflight
-
-    def get_average_duration(self, obj):
-        """Plan.average_duration is an expensive query,
-        so we prefer the already calculated value if available."""
```

Comment:
Maybe we want to move this comment up to where `calculated_average_duration` is sourced?

Reply:
Fixed in c71d0763d6da775ca8db38e2f6c08f32602ec792